- name: Reproduce ouptuts with JDLL 0.5.5-SNAPSHOT (tensorflow_saved_model_bundle)
  status: failed
  error: "java.lang.IllegalArgumentException: Processing method 'scale_range' failed\
    \ throwing an InvocationTargetException. Go to the following link to see valid\
    \ transformations:https://github.com/bioimage-io/core-bioimage-io-java/tree/master/src/main/java/io/bioimage/specification/transformation.\
    \ \n\njava.lang.IllegalArgumentException: At the moment, only allowed scaling\
    \ of planes."
  traceback: "java.lang.IllegalArgumentException: Processing method 'scale_range'\
    \ failed throwing an InvocationTargetException. Go to the following link to see\
    \ valid transformations:https://github.com/bioimage-io/core-bioimage-io-java/tree/master/src/main/java/io/bioimage/specification/transformation.\
    \ \n\njava.lang.IllegalArgumentException: At the moment, only allowed scaling\
    \ of planes."
  source_name: /home/runner/work/icy-bioimage-io/icy-bioimage-io/bioimageio-gh-pages/rdfs/10.5281/zenodo.8421755/8432366/rdf.yaml
  JDLL_VERSION: 0.5.5-SNAPSHOT
